pkgs/tools/virtualization/guestfs-tools/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, fetchurl
+, bison
+, cdrkit
+, cpio
+, flex
+, getopt
+, hivex
+, jansson
+, libguestfs
+, libvirt
+, libxml2
+, makeWrapper
+, ncurses
+, ocamlPackages
+, pcre2
+, perlPackages
+, pkg-config
+, qemu
+, xz
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guestfs-tools";
+  version = "1.48.2";
+
+  nativeBuildInputs = [
+    bison
+    cdrkit
+    cpio
+    flex
+    getopt
+    makeWrapper
+    pkg-config
+    qemu
+  ] ++
+  (with perlPackages; [
+    GetoptLong
+    libintl-perl
+    ModuleBuild
+    perl
+    Po4a
+  ]) ++
+  (with ocamlPackages; [
+    findlib
+    gettext-stub
+    ocaml
+    ocaml_gettext
+    ounit2
+  ]);
+
+  buildInputs = [
+    hivex
+    jansson
+    libguestfs
+    libvirt
+    libxml2
+    ncurses.dev
+    pcre2.dev
+    xz.dev
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/virt-win-reg \
+      --prefix PERL5LIB : ${with perlPackages; makeFullPerlPath [ hivex libintl-perl libguestfs ]}
+  '';
+
+  enableParallelBuilding = true;
+
```

Comment:
This is the default.
```suggestion
```

Reply:
Yes, just tested this out and it indeed doesn't parallelize builds by default. This seems wildly inefficient and confusing to have it disabled by default and not just per-package basis.

Reply:
They tried to enable this by default a few years ago, and it basically broke so many packages that they decided to just disable it again.

Reply:
This is the default for cmake and I think meson.